playbooks/inventory.yaml

```diff
@@ -103,23 +103,24 @@
 
```

Comment:
We use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format (with footer) for ALL commit messages, please fix your commit message.

within the commits foote close #34.

_If not known, since you have a single commit only, you can use `git commit --amend` to fix the commit message. Then you'll have to use `git push --force` to update the remote_